### ISSUE TITLE
DBForBix don't retry connection on a DB after an outage

### ISSUE THREAD

**Issue by menardorama** [smartmarmot/DBforBIX#13]
Hi 
I''m testing dbforbix since 2 weeks now
One issue I see (same in 0.6) is that when you have an outage on a monitored database, DBForbix don't  try to reconnect .

I get this output : 
```
org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:750)
	at org.postgresql.jdbc.PgConnection.clearWarnings(PgConnection.java:679)
	at org.apache.commons.dbcp2.DelegatingConnection.clearWarnings(DelegatingConnection.java:357)
	at org.apache.commons.dbcp2.datasources.InstanceKeyDataSource.getConnection(InstanceKeyDataSource.java:954)
	at org.apache.commons.dbcp2.datasources.InstanceKeyDataSource.getConnection(InstanceKeyDataSource.java:859)
	at com.smartmarmot.dbforbix.db.adapter.AbstractAdapter.getConnection(AbstractAdapter.java:65)
	at com.smartmarmot.dbforbix.db.adapter.PGSQL.getConnection(PGSQL.java:23)
	at com.smartmarmot.dbforbix.scheduler.Scheduler.run(Scheduler.java:92)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
2016-12-02 14:41:05 ERROR This connection has been closed.
```

I think that having a reconnection would be needed; most people will forget to restart dbforbix.

**Comment by askifyouneed** [comment]
I confirm. The same problem is with the Oracle, that restart service is necessary.

**Comment by smartmarmot** [comment]
This is actually managed by Apache JDBC and the Oracle driver registered in the pool. and I found the bug... https://issues.apache.org/jira/browse/DBCP-442 I'll update and fix Unfortunately this bug is not that much constant... I took a look at my setup and I've spotted this issue from time to time but not systematically.

**Comment by smartmarmot** [comment]
Fixed in 2.2
